projects/fsastorefrontstyles/scss/overrides/main-overrides.scss

```diff
@@ -75,7 +75,8 @@ cx-fs-cms-custom-container {
 }
 // Temporary solution, untill forms are wrapped in different container component
 .CheckoutPageTemplate .BottomContent cx-fs-cms-custom-container,
-.MultiStepCheckoutSummaryPageTemplate .BodyContent cx-fs-cms-custom-container {
+.MultiStepCheckoutSummaryPageTemplate .BodyContent cx-fs-cms-custom-container,
+.MultiStepCheckoutSummaryPageTemplate .SideContent cx-fs-cms-custom-container {
```

Comment:
If possible we should move this up in this file, since we already have `.MultiStepCheckoutSummaryPageTemplate` selector here. That would avoid this duplicating.

Reply:
Yes, but then we'll have the same snippet of code in multiple places. And what if in the future we have a need for these styles on the different template pages, to move the same styles in that template and duplicate the code? Actually, it is the case now, we have a cms container in the `CheckoutPageTemplate` and `MultiStepCheckoutSummaryPageTemplate`. So instead to extract styles in one place, you think it is better to separate that and duplicate the code?

Reply:
However you look at it it's duplicated. Whether it's duplicated selectors (like it is in this case here) or duplicated CSS rules. But I would agree that maybe duplicated selectors are lesser evil. Can we then write this in some sort of common selector?